CHANGELOG.md

```diff
@@ -1,11 +1,49 @@
-## Next release
-
-This release includes changes to amtool which are not fully backwards compatible with the previous
-amtool version (#1798) related to backup and import of silences.  If a backup of silences is created using 
-a previous version of amtool (v0.16.1 or earlier), it is possible that not all silences can be correctly imported using a later version of amtool.
-
+## 0.17.0 / 2019-05-02
+
+This release includes changes to amtool which are not fully backwards
+compatible with the previous amtool version (#1798) related to backup and
+import of silences. If a backup of silences is created using a previous
+version of amtool (v0.16.1 or earlier), it is possible that not all silences
+can be correctly imported using a later version of amtool.
+
+Additionally, the groups endpoint that was dropped from api v1 has been added
+to api v2. The default for viewing alerts in the UI now consumes from this
+endpoint and displays alerts grouped according to the groups defined in the
+running configuration. Custom grouping is still supported.
+
+This release has added two new flags that may need to be tweaked. For people
+running with a lot of concurrent requests, consider increasing the value of
+`--web.get-concurrency`. An increase in 503 errors indicates that the request
+rate is exceeding the number of currently available workers. The other new
+flag, `--web.timeout`, may also need modification from its default value.
```

Comment:
Default value is "no timeout at all". Perhaps reword the last sentence as:

"The other new flag, `--web.timeout`, limits the time a request is allowed to run. The default behavior is to not use a timeout.